Lib/test/test_concurrent_futures.py

```diff
@@ -55,6 +56,15 @@ def my_method(self):
         pass
 
 
+class EventfulGCObj():
+    def __init__(self, ctx):
+        mgr = get_context(ctx).Manager()
```

Comment:
What is the rationale for using a manager here? Since the object is instantiated in the parent, it should be inheritable by the child anyway.

Reply:
The executor is launched in the `setup` method of the TestCase. Thus, there is no possibility to pass the `Event` object thru inheritence and the job `(id, obj)` is passed via pickle.

Reply:
Oh, right. I've never used managers and I was surprised to see this...